src/app/shared/components/wednesday-frog/wednesday-frog.component.html

```
@@ -1,4 +1,4 @@
-<div class="mt-5 d-md-flex justify-content-end" *ngIf="showFrog">
+<div class="mt-3 md-sm-0 d-md-flex justify-content-end" *ngIf="showFrog">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)